src/core/account/account-api.ts

```diff
@@ -503,6 +504,29 @@ export function makeAccountApi(ai: ApiInput, accountId: string): EdgeAccount {
       return await finishWalletCreation(ai, accountId, walletInfo.id, opts)
     },
 
+    async createCurrencyWallets(
+      createWallets: EdgeCreateCurrencyWallet[]
+    ): Promise<EdgeCurrencyWallet[]> {
+      const { login, loginTree } = accountState()
+
+      const walletInfos = await Promise.all(
+        createWallets.map(async opts => {
+          return await makeCurrencyWalletKeys(ai, opts.walletType, opts)
+        })
+      )
+      await applyKit(ai, loginTree, makeKeysKit(ai, login, walletInfos))
+      return await Promise.all(
```

Comment:
In a situation where some wallets are created and some aren't (failure for whatever reason) should this function still return an array of wallets and errors or throw if there's just a single error? The successful ones would still be present in the account even if this function doesn't return, presumably.